fs/rc/internal_test.go

```diff
@@ -119,3 +119,21 @@ func TestCoreQuit(t *testing.T) {
 	_, err := call.Fn(context.Background(), in)
 	require.Error(t, err)
 }
+
+// core/command: Runs a raw rclone command
+func TestCoreCommand(t *testing.T) {
+	call := Calls.Get("core/command")
+
+	in := Params{
+		"command": "version",
+		"opt":     map[string]string{},
+		"arg":     []string{},
+	}
+	got, err := call.Fn(context.Background(), in)
+	require.NoError(t, err)
```

Comment:
You could check that the current version `fs.Version` is contained in the output returned from rclone here.

Reply:
Thanks for this :-)